 

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
hmmm not sure what this is, but seems like an empty file with a new line?

Reply:
Same. My diff for this commit shows it as `/dev/null` before, and `b/` after. I may need to cherrypick changes

Reply:
Fixed!